### ISSUE TITLE
cDock crashes the dock in macOS Ventura 13

### ISSUE THREAD

**Issue by suishouen** [w0lfschild/cDock2#335]
cDock 5.0.2 works partially in macOS Monterey 12.
But cDock crashes the dock in macOS Ventura 13.

Would you have any plan to update cDock?

**Comment by devmistry** [comment]
I second this! 

It worked just about right with Monterey :(

**Comment by mdilling** [comment]
I echo the same as the first two commentors.

**Comment by r3volution11** [comment]
I've yet to install Ventura until my hope for this to happen is exhausted.

**Comment by devmistry** [comment]
@w0lfschild - i can understand that you may practically given up on the project, but I'd like to just nudge and remind you, there are SO many people who LOVE cDock. It's unique and does EXACTLY what we want. If there's anything you can do - even as a private release for us -we would ever be so grateful. And BTW, I believe some of us also paid-upgraded to v5 when it was in BETA. Please please please - just let us know if we can continue to hope 😪

**Comment by webxsite** [comment]
Yeah also holding off on installing Ventura and willing to pay for cDock 5

**Comment by driecken** [comment]
cDock doesn't crash on my macOS Ventura but it doesn't really tweak the dock either.. even with the System Integrity Protection disabled, and having Library Validation disabled.. 
I second the views of other users here and would pay for a new license if this app was updated or released in a new version to restore functionality on macOS Ventura.
The dock from macOS 10.10 up to now is very basic and old fashioned, 2D looks really boring compared to the three dimensional glassy dock from macOS 10.5 through macOS 10.9, it has perspective instead of the traditional flat one they use now...

**Comment by r3volution11** [comment]
I'm definitely up for paying for a new version. I haven't updated to Ventura yet and I know there's a v5 beta around that may work but there hasn't been anyone saying so. If it takes some CLI work to get going, please share it with us. I'll still pay. That said, I'll understand why you wouldn't want to put in the effort for the few amount of people willing to pay. But please let us know one way or the other.

For me, I just want the ability to remove the dock's background, reduce the amount of space the dock takes up (while still having decent-sized icons), add spacers, and hide apps from the dock. But I'll be happy with the first two alone. This is my dock: https://r3v.in/Cru9/. It's nothing special but its UI is more productive than the one Apple provides.

**Comment by expresspotato** [comment]
Seems like everyone is willing to pay... We're hoping you will come back to the table @w0lfschild :)

**Comment by beardcc** [comment]
I would pay!

**Comment by etrahretep** [comment]
I already paid twice....
Sad
